docs/Acknowledgement_text.rst

```diff
@@ -0,0 +1,19 @@
+.. note::
+
+   Researchers are requested to acknowledge Mila when publications or reports
+   are published on research that used resources provided by the organization.
+   Our ability to showcase the successful use of our resources builds an
+   important case for long-term and sustainable funding of Mila and our
+   operations. Examples include the use, either in whole or in part, of:
+
+   * Training Models
+   * Exploring Datasets
+   * Computing, visualization or storage hardware
+   * Software
+   * Collaboration facilities
```

Comment:
Does this become too wide and lose some of its significance?

Do we want to make this more about the Mila cluster, or is it fine to have it be a general "yay Mila!" acknowledgement, in which case that's a bit implied by the fact that people are using @mila.quebec in their email addresses?

Reply:
Yes, removed some of the bullets